keras_cv/models/mobilenet_v3.py

```diff
@@ -0,0 +1,363 @@
+"""MobileNet v3 models for KerasCV.
+
+Reference:
+  - [Searching for MobileNetV3](https://arxiv.org/pdf/1905.02244.pdf) (ICCV 2019)
+  - [Based on the Original keras.applications MobileNetv3](https://github.com/keras-team/keras/blob/master/keras/applications/mobilenet_v3.py)
+"""
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import backend
+from tensorflow.keras import layers
+
+channel_axis = -1
+
+
+def Depth(divisor=8, min_value=None, name=None):
+    if name is None:
+        name = f"depth_{backend.get_uid('depth')}"
+
+    def apply(x):
+        nonlocal min_value
+        if min_value is None:
+            min_value = divisor
+        new_x = max(min_value, int(x + divisor / 2) // divisor * divisor)
+        # Make sure that round down does not go down by more than 10%.
+        if new_x < 0.9 * x:
+            new_x += divisor
+        return new_x
+
+    return apply
+
+
+def HardSigmoid(name=None):
+    if name is None:
+        name = f"hard_sigmoid_{backend.get_uid('hard_sigmoid')}"
+
+    def apply(x):
+        return layers.ReLU(6.0)(x + 3.0) * (1.0 / 6.0)
+
+    return apply
+
+
+def HardSwish(name=None):
+    if name is None:
+        name = f"hard_swish_{backend.get_uid('hard_swish')}"
+
+    def apply(x):
+        return layers.Multiply()([x, HardSigmoid()(x)])
+
+    return apply
+
+
+def SEBlock(filters, se_ratio, prefix, name=None):
+    if name is None:
+        name = f"se_block_{backend.get_uid('se_block')}"
+
+    def apply(inputs):
+        x = layers.GlobalAveragePooling2D(
+            keepdims=True, name=prefix + "squeeze_excite/AvgPool"
+        )(inputs)
+        x = layers.Conv2D(
+            Depth()(filters * se_ratio),
+            kernel_size=1,
+            padding="same",
+            name=prefix + "squeeze_excite/Conv",
+        )(x)
+        x = layers.ReLU(name=prefix + "squeeze_excite/Relu")(x)
+        x = layers.Conv2D(
+            filters,
+            kernel_size=1,
+            padding="same",
+            name=prefix + "squeeze_excite/Conv_1",
+        )(x)
+        x = HardSigmoid()(x)
+        x = layers.Multiply(name=prefix + "squeeze_excite/Mul")([inputs, x])
+        return x
+
+    return apply
+
+
+def ReLU(name=None):
```

Comment:
you can use the built in ReLU instead

Reply:
I forgot to delete this part, was just doing some testing. All places in the code use `layers.ReLU` and not this. So I will just delete this